operator/cmd/syncclusterconfig/sync.go

```diff
@@ -22,6 +22,8 @@ import (
 	"context"
 	"errors"
 	"fmt"
+	"github.com/redpanda-data/redpanda-operator/operator/pkg/admin"
+	"maps"
```

Comment:
NIT: Linter should complain about it.

If you can perform
```
nix develop -c task lint-fix
```

I think it should reorder imports.